src/insns/load_tag_perms.adoc

```diff
@@ -3,9 +3,7 @@ The tag value written to `cd` is 0 if the tag of the memory location loaded is
 0 or the authorizing capability (<<ddc>> or `cs1`) does not grant <<c_perm>>.
 +
 If the authorizing capability does not grant <<lm_perm>>, and the tag of `cd` is 1 and `cd` is not sealed, then an implicit <<ACPERM>> clearing <<w_perm>> and <<lm_perm>> is performed to obtain the intermediate permissions on `cd`.
-+
-If the authorizing capability does not grant <<el_perm>>, and the tag of `cd` is 1, then an implicit <<ACPERM>> clearing <<el_perm>> and restricting the <<section_cap_level>> to the level of the authorizing capability is performed to obtain the final permissions on `cd`.
-+
+++
```

Comment:
This broke the formatting here. We've lost a paragraph break.

![image](https://github.com/user-attachments/assets/207d0365-9608-4c94-a87d-4cf561c37118)

Reply:
ahhh